apps/web/src/app/(private)/datasets/new/page.tsx

```diff
@@ -0,0 +1,69 @@
+'use client'
+
+import {
+  Button,
+  CloseTrigger,
+  DropzoneInput,
+  FormWrapper,
+  Input,
+  Modal,
+  // useToast,
+} from '@latitude-data/web-ui'
+// import { useFormAction } from '$/hooks/useFormAction'
+import { useNavigate } from '$/hooks/useNavigate'
+import { ROUTES } from '$/services/routes'
+
+// import useProjects from '$/stores/projects'
+
+export default function NewDataset() {
+  const data = { name: '' }
+  const navigate = useNavigate()
+  // const { toast } = useToast()
+  // const { create } = useProjects()
+  // const { data, action } = useFormAction(create, {
+  //   onSuccess: (project) => {
+  //     toast({
+  //       title: 'Success',
+  //       description: `Project "${project.name}" created.`,
+  //     })
+  //
+  //     navigate.push(ROUTES.datasets.root)
+  //   },
+  // })
```

Comment:
??

Reply:
Lazy, I want to have it here for when I do my own action